07.ls_object/lib/ls_object.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require 'pathname'
+require_relative 'ls_file_stat'
+require_relative 'screen'
+
+class LsObject
+  def initialize(paths, options)
+    @paths = paths.empty? ? ['.'] : paths
+    @options = options
+  end
+
+  def main
+    @options[:header] = !(@paths.length == 1 || @paths.empty?)
```

Comment:
```suggestion
    @options[:header] = @paths.length > 1
```

でいいのでは？

あと、mainメソッドを呼び出したら`@options`が書き替えられる、とするよりもinitializeの中でheaderも設定しておいて、それ以降の処理では`@options`は変化しない、とした方がいいですね